libs/orbit-essentials/src/http.rs

```diff
@@ -61,3 +61,37 @@ fn cbor_encode(value: &impl Serialize) -> Vec<u8> {
 pub fn certified_data_for_skip_certification() -> [u8; 32] {
     skip_certification_asset_tree().digest()
 }
+
+pub fn not_found() -> HttpResponse {
+    HttpResponse {
+        status_code: 404,
+        headers: vec![HeaderField("Content-Type".into(), "text/plain".into())],
+        body: "404 Not Found".as_bytes().to_owned(),
+    }
+}
+
+pub fn parse_path(url: &str) -> Option<&str> {
```

Comment:
Why are you not using a library for this?

Reply:
We discussed this with @keplervital and concluded it's unnecessary to include a library for such a simple and isolated use case where only a dev environment is affected. WDYT?

Reply:
> are you sure that none of the existing dependencies can deal with that?
Good call, I'm not. I'll investigate.

Reply:
I see, thank you for looking into this!